src/types/types.ts

```diff
@@ -1,9 +1,12 @@
 import { Property } from '@/components/Property/Property';
+
 import { PropertyVisibilityNumber } from './enums';
 
+/*--------------------------------------------------------------------------------------*/
```

Comment:
Hm, not a fan of these dividers, and don't fully agree with their placement either.  

If we have dividers, it should be made clear what type of types belong in each section

Personally, I would prefer to just remove them, I think